db/model/Gdoc/Gdoc.ts

```diff
@@ -413,6 +457,17 @@ export class Gdoc extends BaseEntity implements OwidGdocInterface {
         return gdoc
     }
 
+    static async getDetailsOnDemandGdoc(): Promise<
+        DetailDictionary | undefined
+    > {
+        const gdoc = await Gdoc.getGdocFromContentSource(
```

Comment:
This should have an error handler if `GDOCS_DETAILS_ON_DEMAND_ID` is undefined

Reply:
Also double check what causes the error when adding this document the first time

Reply:
And remove the old details button from the sidebar